packages/TelegramClient-UI.package/TCUPhotoMessage.class/instance/addPhoto.st

```diff
@@ -0,0 +1,14 @@
+constructing
+addPhoto
+
+	| photoForm |
+	photoForm := self messageModel core imageStore getFormFor: self messageModel fileId.
+	
+	"To be compatible with Squeak 5.2 we can't use Form >> scaledToWidth. 
+	photoForm := photoForm scaledToWidth: self class defaultWidth. is equivalent to the line below."
+	photoForm := photoForm magnify: photoForm boundingBox by: (self class defaultWidth / photoForm width) smoothing: 2.
```

Comment:
I would seriously consider dropping support for 5.2, so many missing UI features...

Reply:
I would leave it as is for now bcs this should be a different PR

Reply:
yes, definitely